lib/AST/ASTStructuralEquivalence.cpp

```diff
@@ -1633,6 +1742,8 @@ unsigned StructuralEquivalenceContext::getApplicableDiagnostic(
     return diag::warn_odr_parameter_pack_non_pack;
   case diag::err_odr_non_type_parameter_type_inconsistent:
     return diag::warn_odr_non_type_parameter_type_inconsistent;
+  default:
```

Comment:
This is probably not related too.

Reply:
I wanted to make a build warning go away (I have another one in ASTImporterTest.cpp, probably a new PR can be created to correct this).